versions/1/data_stream/_dev/spec.yml

```diff
@@ -5,4 +5,9 @@ spec:
       type: folder
       name: test
       required: false
-      $ref: "./test/spec.yml"
\ No newline at end of file
+      $ref: "./test/spec.yml"
+    - description: Folder containing configuration related to deploying the package's integration service(s)
+      type: folder
+      name: deploy
+      required: false
+      $ref: "./../../_dev/deploy/spec.yml" # Use "deploy" spec for package
```

Comment:
Is this what we want? It implies that there can be data stream level deployment definitions for Docker Compose-based deployments as well, not just Terraform-based deployments. Having such consistency is really nice but I want to make sure it's something that's actually going to be supported by `elastic-package` as well. Otherwise the spec is a bit misleading.

Reply:
Yes, I modified the `elastic-package` here: https://github.com/elastic/elastic-package/pull/228 . 

As you noticed, It means that we will support deploy definitions on the data stream level. Actually we may leverage from this change, as currently developers build a single Docker image for multiple data streams. See: https://github.com/elastic/integrations/tree/master/packages/zeek/_dev/deploy/docker

If you prefer to support `tf` only, I can adjust the spec.

Reply:
Ah nice, I hadn't studied https://github.com/elastic/elastic-package/pull/228 in depth before, sorry! Given what you have there, what you have here is good as-is. Thanks!